packages/pnet/src/errors.ts

```diff
@@ -3,3 +3,4 @@ export const INVALID_PSK = 'Your private shared key is invalid'
 export const NO_LOCAL_ID = 'No local private key provided'
 export const NO_HANDSHAKE_CONNECTION = 'No connection for the handshake provided'
 export const STREAM_ENDED = 'Stream ended prematurely'
+export const ERR_INVALID_PARAMETERS = 'ERR_INVALID_PARAMETERS'
```

Comment:
should this and others be in `@libp2p/interface` ?

I know we had a discussion about this somewhere..

Reply:
Yes, some generic codes will reduce duplication but no need to block on this.